### ISSUE TITLE
XNNPACK operators are not actually registered under xnnpack namespace

### ISSUE THREAD

**Issue by ezyang** [pytorch/pytorch#36517]
Here's the registration line:

```
        .op("prepacked::linear_clamp_prepack(Tensor W, Tensor? B=None, "
            "Scalar? output_min=None, Scalar? output_max=None) "
            "-> __torch__.torch.classes.xnnpack.LinearOpContext",
            torch::RegisterOperators::options()
            .aliasAnalysis(at::AliasAnalysisKind::PURE_FUNCTION)
            .kernel<decltype(createLinearClampPrePackOpContext),
                createLinearClampPrePackOpContext>(
                    DispatchKey::CPU))
```

Doesn't look like xnnpack namespace to me.

cc @kimishpatel

**Comment by ezyang** [comment]
The comment above implies that it should be registered in xnnpack namespace:

```
  // Registering under _xnnpack namespace for now. As we add more backend requiring similar functionality
  // We can refactor the code and use a better namespace.
```

**Comment by kimishpatel** [comment]
My bad. Comment is outdated. I will remove.

**Comment by ezyang** [comment]
@kimishpatel in that case, I have an inverse question: the torchbind classes LinearOpContext and Conv2dOpContext are still in the xnnpack namespace. Should they be in the prepack namespace now?

**Comment by kimishpatel** [comment]
@ezyang, yes. The state of code at this point is stuck somewhere in the middle. Discussing with @dzhulgakov, @dreiss and @AshkanAliabadi we decided that we did not want op registered under xnnpack namespace since these particular ops can capture a generic behavior of the stateful ops that are not backend specific. So it can be xnnpack or mobile_gpu for example. However the actual namespace and directory structure was not changed as part of the PR (https://github.com/pytorch/pytorch/pull/35010/) that moved it from xnnpack to prepack op namespace.
I think we can keep this issue open until all of that is consistent.
